h/exceptions.py

```diff
@@ -3,3 +3,10 @@ class InvalidUserId(Exception):
 
     def __init__(self, user_id):
         super().__init__(f"User id '{user_id}' is not valid")
+
+
+class RealtimeMessageQueueError(Exception):
+    """A message could not be sent to the realtime Rabbit queue."""
+
+    def __init__(self):
+        super().__init__("Could not queue message")
```

Comment:
:+1: The use of a custom exception class should make this particularly easy to search for in Sentry too